serverless-heaven/serverless-webpack
# New: Patch for #570

## What did you implement:
Recently I ran into the problem described in #299 and after testing several possible solutions (see [this issue comment](https://github.com/serverless-heaven/serverless-webpack/issues/299#issuecomment-731618090) for a more detailed rundown of what I tried) I ended up using the fix implemented in #570. However, I ended up having some problems with the watch command used by serverless-offline. This PR implements a patch that fixes those issues and gets it working properly.

Closes #299



## How did you implement it:
I made the script that is responsible for the watch command compatible with the rest of the changes made in the PR. Note that I didn't make the watch command use worker-farm for compilation when watching the changes, as it currently works fine without it.

## How can we verify it:
1. Verify that #570 works fine (this PR depends on it)
2. Setup a new project that uses serverless-offline and try to spin a new local version with this PR's code, it should work fine

## Todos:


***Is this ready for review?:*** Yes
***Is it a breaking change?:*** NO for #570, YES for the current package
